src/screens/Requests/Requests.tsx

```diff
@@ -372,6 +418,11 @@ const Requests = (): JSX.Element => {
             </Table>
           </InfiniteScroll>
         )}
+        {!isLoading && displayedUsers.length === 0 && (
+          <div className="w-100 text-center my-4">
+            <h5 className="m-0 ">{t('noRequestFound')}</h5>
```

Comment:
Why remove this line ?

```js
{t('noResultsFoundFor')} &quot;{searchByName}&quot;
```

If no request is found for a particular search, the above text should be displayed